src/storage/metrics.rs

```diff
@@ -100,4 +92,29 @@ lazy_static! {
         &["type"],
         exponential_buckets(1.0, 2.0, 21).unwrap()
     ).unwrap();
+    pub static ref KV_GC_EMPTY_RANGE_COUNTER: IntCounter = register_int_counter!(
```

Comment:
The naming of these statics seems to not be very consistent.

Reply:
Since it was no longer a command of Scheduler, I removed the word "COMMAND"...